nutrition_labels/predict.py

```diff
@@ -0,0 +1,90 @@
+"""
+For a dataset load a model (or ensemble of models) and predict whether the grants
+are tech grants or not.
+
+Run as:
+python nutrition_labels/predict.py --config_path configs/predict/2021.04.02.ini
+
+"""
+
+from argparse import ArgumentParser
+import configparser
+import os
+
+import pandas as pd
+
+from nutrition_labels.grant_tagger import GrantTagger
+from nutrition_labels.ensemble_grant_tagger import EnsembleGrantTagger
+
+def output_tagged_grants(output_path, y_pred, grant_ids):
+
+    if not os.path.exists(os.path.dirname(output_path)):
+        os.makedirs(os.path.dirname(output_path))
+
+    pd.DataFrame(
+        {
+        'Tech grant prediction': y_pred,
+        'Grant ID': grant_ids
+        }
+        ).to_csv(output_path, index=False)
+
+def predict_grants(config):
+
+    grants_data_path = config["prediction_data"]["grants_data_path"] 
```

Comment:
See comment above on decoupling. I think it makes a bit more sense to me if this function receives a list of grants instead. Then you can open the file and pass the list to it, but I could also just use it as is in a pipeline.